Update documentation (#6)
MiaAltieri/mongodb-operator: README.md

> # MongoDB Operator

Reviewer: ```suggestion
# Charmed MongoDB Operator
```

## Description

> The MongoDB Operator provides a document database using [MongoDB](https://www.mongodb.com/).

Reviewer: ```suggestion
The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database.
```

> MongoDB is a general purpose distributed document database. This charm deploys and operates MongoDB on a machine cloud.

Reviewer: ```suggestion
MongoDB is a general purpose distributed document database. This operator charm deploys and operates MongoDB on physical or virtual machines.
```

> This charm seeks to migrate the features from the current Mongo charm that uses the Reactive framework to the new Operator framework.

Reviewer: ```suggestion
```

## Usage

> As this charm is not yet published, you need to follow the build and deploy instructions from [CONTRIBUTING.md](./CONTRIBUTING.md).

Reviewer: ```suggestion
As this operator charm is not yet published, you need to follow the build & deploy instructions from [CONTRIBUTING.md](./CONTRIBUTING.md).
```

## Relations

> Supported relations:

Reviewer: ```suggestion
Supported [relations](https://juju.is/docs/olm/relations):
```
